### PR TITLE
Get XMPP Host from Bundle Identifier

### PR DESCRIPTION
**Pull request by pmurphyjam**
Submitting pull request for new XMPP Host from Bundle Identifier
BugzID: 410

### CODE REVIEW

File: Core/XMPPStream.m [hunk 1]
```diff
@@ -3760,14 +3757,30 @@ - (void)tryNextSrvResult
...
+
 	while (srvResultsIndex < [srvResults count])
 	{
 		XMPPSRVRecord *srvRecord = [srvResults objectAtIndex:srvResultsIndex];
 		NSString *srvHost = srvRecord.target;
 		UInt16 srvPort    = srvRecord.port;
 		
-		//success = [self connectToHost:srvHost onPort:srvPort withTimeout:XMPPStreamTimeoutNone error:&connectError];
-		success = [self connectToHost:XMPP_HOST onPort:5222 withTimeout:XMPPStreamTimeoutNone error:&connectError];
+        success = [self connectToHost:xmppHostName onPort:5222 withTimeout:XMPPStreamTimeoutNone error:&connectError];
```
Comment (ryan-medigram) [inline Core/XMPPStream.m]: This line looks like it's missing an indent - could you add one?

File: Core/XMPPStream.m [hunk 2]
```diff
@@ -3789,7 +3802,7 @@ - (void)tryNextSrvResult
 		// 
 		// In other words, just try connecting to the domain specified in the JID.
 		
-		success = [self connectToHost:[myJID_setByClient domain] onPort:5222 withTimeout:XMPPStreamTimeoutNone error:&connectError];
+        success = [self connectToHost:xmppHostName onPort:5222 withTimeout:XMPPStreamTimeoutNone error:&connectError];
```
Comment (ryan-medigram) [inline Core/XMPPStream.m]: Missing indent here as well, but otherwise this PR looks great!

Comment (brettrcooper2008) [inline Core/XMPPStream.m]: I agree!  (wanna unsubscribe me?  _wink_)

**Comment by ryan-medigram** [comment]
@brettrcooper2008 will do! gimme a few minutes.

**Comment by ryan-medigram** [comment]
@pmurphyjam Thanks for making those changes! I'll merge this in shortly.

**Pull request merged**